apps/dataset-browser/tailwind.config.ts

```diff
@@ -0,0 +1,7 @@
+import sharedConfig from 'tailwind-config/tailwind.config';
```

Comment:
1. The name `tailwind-config/tailwind.config` sounds a bit redundant. Is is possible to use e.g. just `from 'tailwind-config'`?
2. The names `sharedConfig` and `config` sound a bit confusing to me; they're quite similar. Would the following work (untested)?

```
import config from 'tailwind-config/tailwind.config';

export default {
  presets: [config],
};
```